tests/routes/metadata_test.py

```diff
@@ -31,6 +31,16 @@ def test_put_metadata(self):
             .put(data=meta_utils.send_meta(), headers=self.auth)
         self.assert_metadata_matches("/test/artifact/dir/dir2/dir3/nest-test/_meta")
 
+    def test_put_metadata_invalid_request_data(self):
+        self.route_tester \
+            .metadata() \
+            .route_params(bucket_name="test", path="dir/dir2/dir3/nest-test") \
+            .expect(400, {
+                "code": "invalid_request_data_format",
+                "message": "{u'notValue': u'biluga'} is not of additionalProperties False"
```

Comment:
Can we json dump the value before logging it?  This would be confusing to anyone not a python dev